vulkaninfo/vulkaninfo.cpp

```diff
@@ -210,14 +210,14 @@ void DumpPresentableSurfaces(Printer &p, AppInstance &inst, const std::vector<st
             auto exists = surface_list.end();
             for (auto it = surface_list.begin(); it != surface_list.end(); it++) {
                 // use custom comparator to check if the surface has the same values
-                if (it->gpu == gpu.get() && *it->surface == *surface.get()) {
+                if (it->gpu == gpu.get() && *(it->surface) == *(surface.get())) {
                     exists = it;
```

Comment:
We can `break` after we find the first (and should be the only) match, yes?

Reply:
Yes, that is a good point. It should be the only match, but better to be safe than assume.